packages/react-components/react-carousel-preview/library/src/components/CarouselNav/CarouselNav.types.ts

```diff
@@ -13,6 +13,7 @@ export type NavButtonRenderFunction = (index: number) => React.ReactNode;
 export type CarouselNavState = ComponentState<CarouselNavSlots> & {
   /**
    * The total number of slides available.
+   * Can be overwritten by passing in as a prop.
```

Comment:
I think it'd be helpful to add more information about what overriding the total number of slides would do. When and how would I use the prop?